packages/craco-esbuild/package.json

```diff
@@ -9,15 +9,15 @@
     "src"
   ],
   "peerDependencies": {
-    "@craco/craco": "^6.0.0 || ^7.0.0 || ^7.0.0-alpha",
+    "@craco/craco": "^^7.0.0 || ^7.0.0-alpha",
```

Comment:
let's remove the alpha option there, if people use the latest version they should upgrade to craco stable